php/utils.php

```diff
@@ -559,11 +560,11 @@ function make_progress_bar( $message, $count, $interval = 100 ) {
  * Additionally, this adds 'http://' to the URL if no scheme was found.
  *
  * @param string $url           The URL to parse.
```

Comment:
Alignment is off for this comment block.

Reply:
alignment is the same at is.

Reply:
<img width="611" alt="Image 2020-01-29 at 8 29 12 AM" src="https://user-images.githubusercontent.com/83631/73336255-9a8eaa00-4271-11ea-8429-3b0c35653f5f.png">

Reply:
Issue fixed.